cmd/server/main.go

```diff
@@ -1,3 +1,41 @@
 package main
 
-func main() {}
+import (
+	"evgen3000/go-musthave-metrics-tpl.git/cmd/server/handlers"
+	"evgen3000/go-musthave-metrics-tpl.git/cmd/server/storage"
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/go-chi/chi/v5"
+)
+
+func runServer(host string, router *chi.Mux) {
+	fmt.Println("Server is running on", host)
+	err := http.ListenAndServe(host, router)
+	if err != nil {
+		panic(err)
```

Comment:
[CHANGE REQUEST] Please don't panic
https://github.com/uber-go/guide/blob/master/style.md#dont-panic

Reply:
Okey, okey, I'll be calmer

Reply:
This is still actual.